src/v2/view-builder/views/IdentifierView.js

```diff
@@ -160,6 +164,21 @@ const Body = BaseForm.extend({
     return newSchemas;
   },
 
+  showCustomFormErrorCallout(error) {
+    if (!error?.responseJSON?.errorSummaryKeys?.includes(CUSTOM_ACCESS_DENIED_KEY)) {
+      return false;
+    }
+
+    const { errorSummary } = error.responseJSON;
+    const options = {
+      type: 'error',
+      content: new HtmlView({ message: errorSummary }),
+    };
+
+    this.showMessages(createCallout(options));
+    return true;
+  },
```

Comment:
curious, where is this new function invoked? I don't see any references of it in this PR.